drivers/gles3/storage/config.cpp

```diff
@@ -28,12 +28,12 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
 /**************************************************************************/
 
+#include "core/config/project_settings.h"
```

Comment:
Move down below, it should be after the other headers

Reply:
This head is required by `rasterizer_gles3.h` but not directly, should be some complex dependencies, I revert this change and leave it for following works.

Reply:
Then that's an issue to solve separately I would say